test/e2e/scalability/density.go

```diff
@@ -196,9 +197,11 @@ func logPodStartupStatus(c clientset.Interface, expectedPods int, observedLabels
 
 // runDensityTest will perform a density test and return the time it took for
 // all pods to start
-func runDensityTest(dtc DensityTestConfig) time.Duration {
+func runDensityTest(dtc DensityTestConfig, testPhaseDurations *timer.TestPhaseTimer) time.Duration {
 	defer GinkgoRecover()
 
+	auxObjectCreationPhase := testPhaseDurations.StartPhase(250, "auxiliary object creation")
```

Comment:
Could you name it something more informative? Like "creation of secrets, configmaps and daemons" or sth.
Even better if you can split it into each of those phases.

Reply:
Done.